deployments/sandbox/environment.py

```diff
@@ -94,7 +96,7 @@ def env() -> Mapping[str, Optional[str]]:
             }
         ),
 
-        'azul_dss_query_prefix': '42',
+        'AZUL_DSS_QUERY_PREFIX': '42',
```

Comment:
I did not think it was necessary to mention this change because it would only be used if someone was interfacing with the DSS.
Azul.config also provides default value `''` if it does not locate the env. var. https://github.com/DataBiosphere/azul/blob/8f15f2020f413379b80a57793651cc54ac7e3d7f/src/azul/__init__.py#L202


If you feel that we should include a section about this change I can totally do so, my goal keep this focused on the steps that we actually need.

Reply:
> because it would only be used if someone was interfacing with the DSS

Not quite. On `develop` the variable is used for TDR as well, despite the name. People may have this set in their personal deployments. If they do, they should remove the variable from the environment, *unless* they plan to interface with a DSS instance in the future, which is unlikely.